src/sentry/static/sentry/app/components/gridEditable/flags.tsx

```diff
@@ -7,4 +7,4 @@
  * 3. Allow drag-to-rearrage columns
  */
 export const FLAG_GRID_RESIZABLE = false;
-export const FLAG_GRID_DRAGGABLE = false;
+export const FLAG_GRID_DRAGGABLE = true;
```

Comment:
@dashed remember to revert this change

Reply:
Shouldn't this flag be removed once dragging is working?

Reply:
I'm going to check with @leedongwei ; I assumed this was a feature-flagged feature.

Reply:
@leedongwei gotcha. Thanks for clarifying 👍